ra_aid/tools/expert.py

```
@@ -184,7 +185,7 @@ def ask_expert(question: str) -> str:
 
     query_parts.extend(["# Question", question])
     query_parts.extend(
-        ["\n # Addidional Requirements", "Do not expand the scope unnecessarily."]
+        ["\n # Addidional Requirements", "**DO NOT OVERTHINK**", "**DO NOT OVERCOMPLICATE**"]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)